src/command-dispatcher.ts

```diff
@@ -0,0 +1,82 @@
+import {  } from "./server";
+
+const {createUsersTable, createUser, createLikedLanguagesTable, 
+    addLikedLanguage, getUsers} = require('./server');
+
+//Enumerator with all the commands available
+enum commandsList {
+    CREATE_USER_TABLE = "newusertable",
+    CREATE_LANGUAGES_TABLE = "newlanguagestable",
+    ADD_USER = "newuser",
+    ADD_LIKED_LANGUAGE = "addlanguage",
+    LIST_USERS = "listusers"
+}
+
+//arguments passed with the command
+const args: string[] = (process.argv.slice(2));
+
+/* Entry point of the program.
+   Processes the commands received and calls
+   server functions
+ */
+function dispatchCommand(command: string[]){
```

Comment:
Just how you declare a parameter type, you can declare a function return type to make it extra explicit, and to make sure typescript isn't assuming some dumb types out of this.
Something like: `function foo(bar: string) : Promise<any>{...}`
As a bonus TS/ESLint will warn you if you're doing something wrong, so it's like a soft static typing!

Reply:
That makes sense. Will change it as well on the next commit